routes/userRoutes.js

```diff
@@ -166,4 +167,35 @@ async function validateFields(fields) {
 	return true;
 }
 
+//Update user role
+router.patch('/:id/role', requireLogin, async (req, res) => {
```

Comment:
Shouldn't only admins be allowed to change a user's role?

Reply:
Done